free-programming-books-tr.md

```diff
@@ -61,13 +61,16 @@
 * [Git 101](https://www.gitbook.com/book/aliozgur/git101/details) - [Ali Özgür](https://github.com/aliozgur)
 
 
+### Go
 ### Go
```

Comment:
```suggestion
### Go

```

Go is repeated twice, please remove one of them but make sure the spacing is correct after!